api/src/test/java/uk/gov/dhsc/htbhf/claimant/converter/NewClaimDTOToClaimRequestConverterTest.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.dhsc.htbhf.claimant.converter;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+import uk.gov.dhsc.htbhf.claimant.entity.EligibilityOverride;
+import uk.gov.dhsc.htbhf.claimant.model.EligibilityOverrideDTO;
+import uk.gov.dhsc.htbhf.claimant.model.NewClaimDTO;
+import uk.gov.dhsc.htbhf.claimant.service.ClaimRequest;
+import uk.gov.dhsc.htbhf.dwp.model.EligibilityOutcome;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static uk.gov.dhsc.htbhf.TestConstants.NO_CHILDREN;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimantTestDataFactory.aValidClaimant;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.NewClaimDTOTestDataFactory.aValidClaimDTOWithEligibilityOverride;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.TestConstants.EXPECTED_DELIVERY_DATE_IN_TWO_MONTHS;
+
+@ExtendWith(MockitoExtension.class)
+class NewClaimDTOToClaimRequestConverterTest {
+
+    @InjectMocks
+    NewClaimDTOToClaimRequestConverter claimRequestConverter;
+
+    @Mock
+    ClaimantDTOToClaimantConverter claimantConverter;
+
+    @Test
+    void shouldConvertNewClaimDTOToClaimRequest() {
+        Claimant claimant = aValidClaimant();
+        given(claimantConverter.convert(any())).willReturn(claimant);
+        NewClaimDTO dto = aValidClaimDTOWithEligibilityOverride(EXPECTED_DELIVERY_DATE_IN_TWO_MONTHS, NO_CHILDREN, EligibilityOutcome.CONFIRMED);
+
+        ClaimRequest claimRequest = claimRequestConverter.convert(dto);
+
+        assertThat(claimRequest).isNotNull();
+        assertThat(claimRequest.getClaimant()).isEqualTo(claimant);
+        assertThat(claimRequest.getDeviceFingerprint()).isEqualTo(dto.getDeviceFingerprint());
+        assertThat(claimRequest.getWebUIVersion()).isEqualTo(dto.getWebUIVersion());
+        assertEligibilityOverride(claimRequest.getEligibilityOverride(), dto.getEligibilityOverride());
+
+    }
+
+    private void assertEligibilityOverride(EligibilityOverride eligibilityOverride, EligibilityOverrideDTO dto) {
+        if (null == dto) {
```

Comment:
We shouldn't have a check like this in the test as we know whether or not the EligibilityOverrideDTO is null or not. 
We should have one test for null, and one for not null.